docs/messaging/notifications.mdx

```diff
@@ -240,7 +241,7 @@ FirebaseMessaging.onMessage.listen((RemoteMessage message) {
             channel.id,
             channel.name,
             channel.description,
-            icon: android?.smallIcon,
+            icon: null,
```

Comment:
Could you explain why `android?.smallIcon` is incorrect? This property exists

Reply:
Hello, I was absent from Github and I was unable to return, so to be able to solve this problem, what happened was that if I left it with `android?.smallIcon` there would be a runtime error when receiving a notification, but the problem was solved, but I I was absent and I couldn't cancel this commit, sorry.